simphony_ui/ui.py

```diff
@@ -218,7 +227,8 @@ def _run_calc_threaded(self):
                 self.update_progress_bar
             )
         except Exception as e:
-            self.calculation_error_event = str(e)
+            tb = traceback.format_exc().splitlines()
```

Comment:
Oh yes that's stupid. I first wanted to remove the last line, that's why I wanted to split the message.